QuietPlace/microphoneListener.js

```diff
@@ -87,74 +90,51 @@ export default class MicrophoneListener extends Component {
     componentWillUnmount() {
         RNSoundLevel.stop()
     }
-	
-	//Volume threshold slider
+
+	//Volume Level Display & Volume Threshold Slider
     render() {
         return (
           <View>
-              <Text style={styles.label}>
-                Volume Threshold{"\n"}
-              </Text>
-              <Text style={styles.volume}>
-                ({Math.trunc(((this.state.value + 160)/160) * 100)}%) {this.state.value} Db
-              </Text>
-              <Slider
-                {...this.props}
-                onValueChange = {value => this.setState({value: value})}
-                style={styles.slider}
-                step = {1}
-                minimumValue = {-160}
-                maximumValue = {0}
-                thumbTintColor = '#C4C4C4'
-                minimumTrackTintColor = 'black'
-                maximumTrackTintColor = 'grey'
-              />
-              <TouchableOpacity style={styles.button}> 
-                <Text style={styles.buttonText}>Settings</Text>
-              </TouchableOpacity>
+            <Speedometer
+              style={styles.speedometer}
+              size={250}
+              value={Math.trunc(((this.soundLevel + 160)/160) * 100)}
+              totalValue={100}
+              innerColor='#8740ad'
+              internalColor='white'
+              outerColor='#9c70b5'
+              showText
+              text={`${this.soundLevel} dB`}
```

Comment:
I get a huge string of decimal places on the current sound level under the speedometer chart. truncating this would be good.

Reply:
Good call! I must have missed that.

I've made the change; it should work properly now.